httpserver/service_node.cpp

```diff
@@ -308,10 +317,14 @@ void ServiceNode::bootstrap_data() {
 
     params["fields"] = fields;
 
-    std::vector<std::pair<std::string, uint16_t>> seed_nodes{
-        {{"storage.seed1.loki.network", 22023},
-         {"storage.seed2.loki.network", 38157},
-         {"imaginary.stream", 38157}}};
+    std::vector<std::pair<std::string, uint16_t>> seed_nodes;
+    if (testnet_) {
+        seed_nodes = {{{"storage.testnetseed1.loki.network", 38157}}};
+    } else {
+        seed_nodes = {{{"storage.seed1.loki.network", 22023},
+                       {"storage.seed2.loki.network", 38157},
+                       {"imaginary.stream", 38157}}};
```

Comment:
As a todo, it would be nice to read those values from an external file :)

Reply:
Not sure if that would be an improvement. Why do you think so?

Reply:
Mainly to separate logic from data.
e.g. when we need to change the seeds, we would not need to touch the code.